packages/jest-enzyme/README.md

```diff
@@ -50,3 +50,9 @@ If you are using Create React App, instead of adding to your `package.json` as a
  ```
 
  This is documented on Create React App at the bottom of the [Testing Components](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#testing-components) section. There is also more information about [Initializing Test Environment](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#initializing-test-environment).
+ 
+ Also, instead of installing the latest version you should stick with versions below 3, because Create React App currently [uses](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/package.json#L50) a version of jest which will cause [this issue](https://github.com/blainekasten/enzyme-matchers/issues/71):
```

Comment:
maybe instead of using verbage like `currently uses` you could say something like.

```If you use a version of create-react-app less then X, it depends on a previous version of jest.. blah blah```